x-pack/plugins/security_solution/cypress/e2e/exceptions/entry/comments.cy.ts

```diff
@@ -0,0 +1,259 @@
+      cy.get(NO_EXCEPTIONS_EXIST_PROMPT).should('not.exist');
+
+      // User 1
+      // open add exception modal
+      addExceptionFlyoutFromViewerHeader();
+
+      cy.get(LOADING_SPINNER).should('not.exist');
+
+      // add exception item conditions
+      addExceptionConditions(getException());
+
+      // add exception item name
+      addExceptionFlyoutItemName('My item name');
+
+      // add exception comment
+      addExceptionComment('User 1 comment');
+
+      // submit
+      submitNewExceptionItem();
+
+      // new exception item displays
+      cy.get(EXCEPTION_ITEM_VIEWER_CONTAINER).should('have.length', 2);
+
+      // click on show comments
+      clickOnShowComments();
+      // copy the first comment to clipboard
+      clickCopyCommentToClipboard();
+
+      cy.get(EXCEPTION_ITEM_COMMENTS_CONTAINER_TEXT).eq(0).should('have.text', 'User 1 comment');
+
+      // User 2
+      // Login with different users to validate accessing comments of different users
+      login(ROLES.detections_admin);
+
+      // Navigate to Rule page
+      visitWithoutDateRange(DETECTIONS_RULE_MANAGEMENT_URL);
+      goToRuleDetails();
+
+      goToRuleDetails();
+      goToExceptionsTab();
+
+      // open edit exception modal
+      openEditException();
+      // add exception comment
+      addExceptionComment('User 2 comment @ using unicode');
+      // submit
+      submitEditedExceptionItem();
+
+      cy.get(EXCEPTION_ITEM_COMMENTS_CONTAINER_TEXT).eq(0).should('have.text', 'User 1 comment');
+
+      cy.get(EXCEPTION_ITEM_COMMENTS_CONTAINER_TEXT)
+        .eq(1)
+        .should('have.text', 'User 2 comment @ using unicode');
+    });
+  });
+
+  describe('Endpoint exceptions', () => {
+    before(() => {
+      esArchiverResetKibana();
+      esArchiverLoad('auditbeat');
+      login();
+      deleteAlertsAndRules();
+      // create rule with exception
+      createEndpointExceptionList().then((response) => {
+        createRule({
+          ...getNewRule(),
+          query: 'event.code:*',
+          index: ['auditbeat*'],
+          exceptions_list: [
+            {
+              id: response.body.id,
+              list_id: response.body.list_id,
+              type: response.body.type,
+              namespace_type: response.body.namespace_type,
+            },
+          ],
+          rule_id: '2',
+        });
+      });
+    });
+
+    beforeEach(() => {
+      visitWithoutDateRange(DETECTIONS_RULE_MANAGEMENT_URL);
+      goToRuleDetails();
+      goToEndpointExceptionsTab();
+    });
+
+    after(() => {
+      esArchiverUnload('auditbeat');
+    });
+
+    it('Add comment on a new exception, and add another comment has unicode character from a different user', () => {
+      // User 1
+      // The Endpoint will populated with predefined fields
+      // when no exceptions exist, empty component shows with action to add exception
+      cy.get(NO_EXCEPTIONS_EXIST_PROMPT).should('exist');
```

Comment:
Is there any reason why this method is here and not in one of the before hooks?

Reply:
Removed it since we are using the esArchiverResetKibana it will make sure that we start on an empty env